packages/core/streams/ai-stream.ts

```diff
@@ -5,110 +5,142 @@ import {
   type ReconnectInterval
 } from 'eventsource-parser'
 
+/**
+ * Interface representing callbacks for an AIStream.
+ * @interface
+ */
 export interface AIStreamCallbacks {
   onStart?: () => Promise<void>
   onCompletion?: (completion: string) => Promise<void>
   onToken?: (token: string) => Promise<void>
 }
 
+/**
+ * Interface representing a custom parser for AIStream data.
```

Comment:
In comments, lets just say what is immediately and remove the `Interface representing...` parts